app/views/tasks/show.html.erb

```diff
@@ -0,0 +1,7 @@
+<h2><%= @task.name%></h2>
+<h3><%=@task.description%></h3>
+<ul>
+  <li><%= link_to "Edit Task", edit_task_path %></li>
+
+
+</ul>
```

Comment:
This is a bad user experience! I can't get back to the index page without the URL!